x/fee_grant/internal/types/key.go

```diff
@@ -0,0 +1,37 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	// ModuleName is the module name constant used in many places
+	ModuleName = "feegrant"
```

Comment:
the `ModuleName` should always match the name of the directory of the module (i.e `fee_grant`). We should change either of them

Reply:
I think there was some issue with underscores in the strings. I adjusted due to a failing test. I can rename the package.

Reply:
Renamed